### ISSUE TITLE
Somgthing wrong between opencv with my blender

### ISSUE THREAD

**Issue by ChapterM** [Fedami/Pix2Blender#1]
Hello, I'm trying to reproduce your code, but I'm encountering some issues between Blender and Python's OpenCV. When I use python.exe in Blender and try to import cv2, I encounter the following problem:
ImportError: DLL load failed while importing cv2
Could you provide me with some assistance? I sincerely appreciate it.

**Comment by Felpami** [comment]
Hello, it is an old university project i don't remember well all the implementation. The error seems related to cv and python version. Have you tried the suggested method described here https://stackoverflow.com/questions/43184887/dll-load-failed-error-when-importing-cv2
Best regards

**Comment by ChapterM** [comment]
> 

Hello! I'm delighted to receive your reply. I browsed the content in the provided link, but I couldn't find a satisfactory solution. I read the documentation of Blender about Bundled Python & Extensions, and following the instructions there, I finally managed to resolve the import issue with cv2.
<img width="711" alt="image" src="https://github.com/Felpami/Pix2Blender/assets/115977434/ab422d17-e54f-4107-b8db-c092ae9ca952">
Thank you very much for your work on Pix2Vox and Blender. I'm glad that I have been able to learn a lot from it.
